### PR TITLE
apps/home: added templates of teasers for overview page

### PR DESCRIPTION
**Pull request by sabinammm**
@fuzzylogic2000 @Rineee  I can't get the teasers with images to display the image. What am I doing wrong in these templates?

### CODE REVIEW

File: apps/home/templates/apps_home/blocks/teaser_image_right.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                <div class="text-center">
+                    {{ value.link.specific.teaser_intro }}
+                    <p>
+                        <br>
+                    </p>
+                    <a href="{{ value.link.url }}" class="btn btn--huge btn-outline-dark">{{ value.link_text }}</a>
+                </div>
+            </div>
+            <div class="col-12 offset-md-1 col-md-4 text-center">
+                <img src="{{ teaser_image.url }}" class="textimg-block-image__img"/>
```
[the remark this answers is not in the document]
Comment (sabinammm) [reply apps/home/templates/apps_home/blocks/teaser_image_right.html:21]: Perfect! will change it and push. Would love to understand this better - I thought that the teaser_image was part of the page model. And I don't understand how this path is created.

[the remark this answers is not in the document]
Comment (fuzzylogic2000) [reply apps/home/templates/apps_home/blocks/teaser_image_right.html:21]: This is still the old way and not showing an image.

File: apps/home/blocks.py [hunk 2]
```diff
@@ -191,6 +191,6 @@ class TeaserBlockImageRight(blocks.StructBlock):
         required=False
     )
 
-    # class Meta:
-    #     template = 'apps_home/blocks/gruenbuch_faq_block.html'
-    #     icon = 'arrow-down'
+    class Meta:
+        template = 'apps_home/blocks/teaser_image_right.html'
+        icon = 'image'
```
Comment (phillimorland) [inline apps/home/blocks.py:196]: This isn't from this pull request and doesn't need to be refactored now but if we need to go over this code again we can actually have just 1 block called aligned image and do like is done [here](https://github.com/liqd/a4-opin/blob/0a759f5419b03b483b1549d0571436f437a0eafc/home/models/blocks.py#L96) in Opin

File: apps/home/templates/apps_home/blocks/teaser_block_two_columns.html [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
+            <div class="col-12 offset-md-1 col-md-4 text-center">
+                <h2 class="cta-ds-block__title">
                     {{ value.link_2.specific.teaser_title }}
                 </h2>
-                <div>
+                <div class="text-center">
                     {{ value.link_2.specific.teaser_intro }}
+                    <p>
+                        <br>
+                    </p>
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply apps/home/templates/apps_home/blocks/teaser_block_two_columns.html:26]: err if you just want space? then add top padding to the container div, or does it do something else?

File: apps/home/templates/apps_home/blocks/teaser_image_right.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (sabinammm) [reply apps/home/templates/apps_home/blocks/teaser_image_right.html:21]: Hmmm, I pushed somewhere else yesterday....sorry about this. It's there now! But still working on philli's change requests.

File: apps/home/blocks.py (hunk 2 above)
Comment (sabinammm) [reply apps/home/blocks.py:185]: Yes, thanks!!

**Pull request merged**